utils/parametric/spec/trace.py

```diff
@@ -70,6 +70,10 @@
 """Key used in the metrics to map to single span sampling max per second."""
 SINGLE_SPAN_SAMPLING_MAX_PER_SEC = "_dd.span_sampling.max_per_second"
 
+SAMPLING_DECISION_MAKER_KEY = "_dd.p.dm"
+SAMPLING_AGENT_PRIORITY_RATE = "_dd.agent_psr"
+SAMPLING_RULE_PRIORITY_RATE = "_dd.rule_psr"
```

Comment:
`_dd.limit_psr` is also relevant, but probing limiter behavior is fraught with flakiness, so let's ignore it for now.